rust/tests/integration/plain_transport.rs

```diff
@@ -190,7 +189,7 @@ fn create_succeeds() {
                 assert!(!transport_dump.direct);
                 assert_eq!(transport_dump.producer_ids, vec![]);
                 assert_eq!(transport_dump.consumer_ids, vec![]);
-                assert_eq!(transport_dump.tuple, transport1.tuple());
+                assert_eq!(transport_dump.tuple.unwrap(), transport1.tuple());
```

Comment:
Yep, it doesn't make much sense to have it optional in a dump, but always present in transport itself. We already have unique types for dumps of different transports.

Reply:
fixed.